opcodes/arc-dis.c

```diff
@@ -779,6 +784,35 @@ parse_option (char *option)
     fprintf (stderr, _("Unrecognised disassembler option: %s\n"), option);
 }
 
+/* Helper for parsing the CPU options.  Accept any of the ARC architecturae
+   values.  */
+
+static unsigned
+parse_cpu_option (char *option)
+{
```

Comment:
It will be nice if we can accept the same CPU names as gas. This will keep the consistency among the tools. Maybe in include/elf/ folder to have something like arc-cpu.def which will be shared between gas and disassm?